models/com.bosch.iot.suite.edge.metric-Filter-1.0.0.type/model.datatype

```diff
@@ -0,0 +1,10 @@
+vortolang 1.0
+namespace com.bosch.iot.suite.edge.metric
+version 1.0.0
+displayname "Filter"
+description "Represents the required metrics per originator."
+
+entity Filter {
+  optional originator as string "The originator identifier."

```

Comment:
It'll be helpful to say what will happen when it's skipped as it's described for the measure identifiers.